### DISCUSSION TITLE
Not sure my input table is good for the analysis

### DISCUSSION THREAD

**Discussion by cleopatraface** [egeulgen/pathfindR#123] [Q&A] [answered]
Hello all,
First, thanks for the code.
second, I have a question. I am working on human brain gene project, so I really don't have a gene expression analysis , but I have a miRNAs expression analysis and I want to use the target genes in an pathway enrichment analysis. I have the log fold for miRNAs. the problem is even if the miRNA is not highly expressed in comparison with the other identified miRNAs, it still have a higher negative regulator effect if his target is highly expressed and targeted by multiple significant miRNAs.  So, my fold change means nothing as all depend on the level of gene expression in the living human which is and will remains for a long time completely unknown.
So O thought about asking your advice. 
Can I use your library anyway ? 
Can I use an NA value for PIN analysis?
And can you give me  any suggestion or advice? :)
Thank you

**Comment by egeulgen** [comment, marked as answer]
Hello,

- fold-change values are only used for visualization purposes
- yes, you can use pathfindR but you will need to use miRNA-miRNA interactions as the PIN, and miRNA gene sets for enrichment
- you cannot omit using a PIN

However, I would suggest using [DIANA tools - mirPath](https://dianalab.e-ce.uth.gr/html/mirpathv3/index.php?r=mirpath) as this was specifically designed for this purpose.

**Comment by cleopatraface** [comment]
Thank you for the advice :)
I already used miRNATap where Diana is included for target prediction. 
Now I have a  filtered list of genes and I want to see the possible pathways and interactions between them
 Can you tell me how to use miRNA-miRNA interactions? what kind of results does it give? does it only shows miRNA regulating other miRNAs?
but thank you for your answer

**Reply by egeulgen** [reply]
Yes miRNA-miRNA interactions would represent miRNAs regulating other miRNAs. In this case, I don't think pathfindR is a suitable choice and I would again suggest using mirPath for enrichment analysis
